### DISCUSSION TITLE
Items from option panel

### DISCUSSION THREAD

**Discussion by felipe-mendieta** [infra-geo-ouverte/igo2#769] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/48095820/165634125-f0aea1aa-85ef-4bf8-99bd-9b8ce3dd5e56.png)

where can I remove one of the options? For example "Advance map tools" or "Catalog", etc. 
In the app demo in **igo2-lib** only delete a mat-list-item and it is all, but in **igo2** i dont know.

**Comment by pelord** [comment, marked as answer]
In the toolbar array in the base.json (or into the context.json itself)

**Reply by pelord** [reply]
https://github.com/infra-geo-ouverte/igo2/blob/6f1e1e16f82e0bba96a48c40985b3f373bb04bf1/src/contexts/_base.json#L163

**Reply by pelord** [reply]
You can also change the tool order.
